### PR TITLE
isk + misc

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 25 functions (+0.26%) on `us`.

✅ This PR fixes 9 warnings!

File: src/effects/sun.c [hunk 1]
```diff
@@ -1,11 +1,60 @@
...
+    data->unk_34 = arg4;
+    data->unk_18 = 255;
+    data->unk_19 = 255;
+    data->unk_1A = 255;
+    data->unk_1B = 255;
+    data->unk_1C = 255;
+    data->unk_1D = 255;
+    data->unk_1E = 255;
+
+    for (i = 0; i < 5; i++) {
```
Comment (ethteck) [inline src/effects/sun.c:48]: ARRAY_COUNT(data->unk_20)

**Review by ethteck — CHANGES_REQUESTED** [review]

**Review by ethteck — APPROVED** [review]

File: src/182B30.c [hunk 2]
```diff
@@ -2059,104 +2059,300 @@ void func_8025D160(ActorPart* arg0, s32 arg1) {
...
+                decor->unk_8BE[decorationIndex] = 0U;
+                angle = (clamp_angle(-part->yaw) * TAU) / 360.0f;
+                sinA = sin_rad(angle);
+                cosA = cos_rad(angle);
+                fx_walking_dust(0,
+                    part->currentPos.x + (part->size.x * sinA * 0.2f),
+                    part->currentPos.y + 1.5f,
+                    part->currentPos.z + (part->size.x * cosA * 0.2f),
+                    sinA, cosA);
+            }
```
Comment (nanaian) [inline src/182B30.c:2256]:
```suggestion
                }
```

Comment (HailSanta) [reply src/182B30.c:2256]: It's aligned with the switch, just looks a bit odd since cases are indented.

Comment (nanaian) [reply src/182B30.c:2256]: Oh, I see. I got confused by the fx_walking_dust call being indented

**Review by nanaian — APPROVED** [review]

**Pull request merged**